test/consensus-gateway/hash_kernel_intent.js

```diff
@@ -19,7 +19,7 @@ const BN = require('bn.js');
 const ConsensusGatewayBase = artifacts.require('ConsensusGatewayBaseTest');
 
 contract('ConsensusGatewayBase::hashKernelIntent', (accounts) => {
-  it('should hash kernel content', async () => {
+  it.skip('should hash kernel content', async () => {
```

Comment:
Why it is skipped ?
We can generate the hash and verify/complete the test.

Reply:
@gulshanvasnani the unit test is comparing against some magic number. I am assuming the contract has been called, value is retrieved and saved. I am not sure that it is giving us a value (just freezing changes to the function, without failing the unit tests). That's why I've skipped it for now. We can also delete it.

Reply:
I think such test case helps in easily/quickly identifying if there is any change in the contract instead identifying it at a later stage.
However, We can discuss it with entire team later on.